pom.xml

```diff
@@ -189,7 +188,7 @@
       <dependency>
         <groupId>commons-codec</groupId>
         <artifactId>commons-codec</artifactId>
-        <version>1.4</version>
```

Comment:
Is this change related to removing commons-httpclient in some way?

Reply:
oh no. going from commons-codec 1.4 to 1.11 was part of a different set of changes I was doing that I must have missed cleaning up. let me push and update that moves that out.

Reply:
Having it the same commit doesn't bother me. I was just curious if there was a connection. Could also update the commit message with a reason for bumping, if there is one.

Reply:
pushed an update that left the commons-codec change out.